src/tests/class_tests/openms/source/ExperimentalDesign_test.cpp

```diff
@@ -65,6 +65,12 @@ START_SECTION(ExperimentalDesign())
 {
   ptr = new ExperimentalDesign();
   TEST_NOT_EQUAL(ptr, null_ptr)
+
+  // Ensure that ExperimentalDesign has a no-except move constructor (otherwise
+  // std::vector is inefficient and will copy instead of move).
+  TEST_EQUAL(noexcept(ExperimentalDesign(std::declval<ExperimentalDesign&&>())), true)
+  TEST_EQUAL(noexcept(ExperimentalDesign::MSFileSection(std::declval<ExperimentalDesign::MSFileSection&&>())), true)
+  TEST_EQUAL(noexcept(ExperimentalDesign::SampleSection(std::declval<ExperimentalDesign::SampleSection&&>())), true)
```

Comment:
ok, I think I can reproduce this here: https://godbolt.org/z/VNob7-

it seems that MSVS cannot create default noexcept move constructors for classes with members using `std::map`, while it works for `std::vector`. I found a stackoverflow post about this regarding std::unordered_map (not quite the same): https://stackoverflow.com/questions/47604029/move-constructors-of-stl-containers-in-msvc-2017-are-not-marked-as-noexcept